client/containers/Pub/PubDocument/PubWordCountButton.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo, useState } from 'react';
+import { Popover, Button, Icon } from '@blueprintjs/core';
+
+import { Node } from 'prosemirror-model';
+
+require('./pubWordCountButton.scss');
+
+type Props = {
+	doc: Node;
+};
+
+const getWordAndCharacterCountsFromDoc = (node: Node) => {
+	const text = node.textBetween(0, node.content.size, ' ', ' ');
+	const words = text.split(' ').filter((word) => word !== '');
+	const wordCount = words.length;
+	const characterCount = words.reduce((a, x) => a + x.length, 0);
```

Comment:
Should spaces count towards character limit? They are omitted here, yes?

Reply:
Good catch. Should definitely count spaces in char count.

Reply:
in which case `const characterCount = node.content.size;`?